package.json

```diff
@@ -27,34 +27,47 @@
   ],
   "scripts": {
     "build": "unbuild",
+    "prepare": "esno ./scripts/vendor.ts",
```

Comment:
Why not jiti 😋

Reply:
I would like to have top-level await landed in jiti 👀

Reply:
Let’s make that happen!